cpp/locality/Voronoi.cc

```diff
@@ -137,9 +138,13 @@ void Voronoi::compute(const freud::locality::NeighborQuery* nq)
                 const vec3<double> normal(normals[3 * neighbor_counter], normals[3 * neighbor_counter + 1],
                                           normals[3 * neighbor_counter + 2]);
 
-                // Ignore bonds in 2D systems that point up or down
-                if (box.is2D() && std::abs(normal.z) > 0)
+                // Ignore bonds in 2D systems that point up or down. This check
+                // should only be dealing with bonds whose normal vectors' z
+                // components are -1, 0, or +1 (within some tolerance).
+                if (box.is2D() && std::abs(normal.z) > 1e-5)
```

Comment:
Are there any cases where the tolerance may differ? This seems like something that should be consistent but I'm curious if there are any special cases.

Reply:
I don’t think there are any special cases - the z component should always be -1, 0, or 1 as noted in the comment above. I looked at the normals from the data data you posted and it seemed to be a valid check.